arch/arm64/src/common/arm64_head.S

```diff
@@ -335,3 +337,37 @@ boot_stage_puts:
 .type boot_low_puts, %function;
 
 #endif /* !CONFIG_ARCH_EARLY_PRINT */
+
+/***************************************************************************
+ * Name: arm64_data_initialize
+ ***************************************************************************/
+
+    .type   arm64_data_initialize, #function
+
+arm64_data_initialize:
+
+    /* Zero BSS */
+
+    adr     x0, .Linitparms
+    ldp     x1, x2, [x0], #8
+
+    mov     x0, #0
+1:
+    cmp     x1, x2
+    bge     2f
+    str     x0, [x1], #8
```

Comment:
Is 8 byte size alignment ensured by a linker script?

Reply:
Done